src/components/editor/inputs/LookupTab.jsx

```diff
@@ -30,12 +30,21 @@ const LookupTab = (props) => {
     )
   }
 
-  if (!props.result.totalHits) {
+  if (props.result.totalHits === 0) {
```

Comment:
Actually, looking at this now, I think all that needs to happen is `if (!props.result.totalHits) { ...}` needs to be placed AFTER `if (props.result.error) { ... }`. Can you try that?

Reply:
Are you sure?  The `totalHits` node missing completely is from a network error, while being  `=== 0` means we got a valid response but there were no results.  Aren't these two separate cases?

Reply:
Doesn't the network error produce an error which is caught and returned in `result.error`? (If not ping me and we can pair on this.)

Reply:
Oh, you are absolutely correct.  Just fixed this up as you suggested.

If there is a network error, you get this message: `NetworkError when attempting to fetch resource.`

Reply:
Note: the diff looks weird, but it's really swapping the order and changing the class for the error message.